File: Python/Enum/src/sample1.py
from enum import Enum


class Color(Enum):
    Red = "The color is red."
    Orange = "The color is orange."
    Yellow = "The color is yellow."
    Green = "The color is green."
    Cyan = "The color is cyan."
    Blue = "The color is blue."
    Purple = "The color is purple."


def main():
    print(Color.Blue)


main()

File: Java/Enum/src/Sample1.java
public class Sample1 {
    enum Color {
        Red("red"),
        Orange("orange"),
        Yellow("yellow"),
        Green("green"),
        Cyan("cyan"),
        Blue("blue"),
        Purple("purple");

        private String raw;

        Color(String raw) {
            this.raw = raw;
        }

        String getRaw() {
            return raw;
        }
    }

    public static void main(String[] args) {
        System.out.println(Color.Blue.getRaw());
    }
}
